test/integration/targets/npm/aliases

```diff
@@ -0,0 +1,3 @@
+posix/ci/group2
+destructive
+skip/freebsd
```

Comment:
Could something be told on why this is skipped on FreeBSD (like a comment ?)

Reply:
I am not sure if `aliases` file could contain any comment, other `aliases` file don't contain any comment. @mattclay ?

Here FreeBSD is skipped because there isn't any FreeBSD tarball there: https://nodejs.org/dist/v9.3.0/.

Reply:
Comments are not yet supported in aliases files. I have that on my list to change though.